src/lib/fragments/Pileup.react.js

```diff
@@ -0,0 +1,96 @@
+import React, {Component} from 'react';
+import {propTypes, defaultProps} from '../components/Pileup.react';
+import pileup from 'pileup';
+import {isNil} from 'ramda';
+
+import '../../../node_modules/pileup/style/pileup.css';
+
+/**
+ * The Pileup component is an genome visualization component
+ * developed by the the Hammerlab. It uses an
+ * example integration of pileup.js and React (https://www.npmjs.com/package/pileup).
+ */
+export default class Pileup extends Component {
+    constructor(props) {
+        super(props);
+        this.ref = React.createRef();
+        this.pileup = null;
+
+        this.parseTracks = this.parseTracks.bind(this);
```

Comment:
```suggestion
        this.parseTracks = this.parseTracks.bind(this);
        this.createPileupBrowser = this.createPileupBrowser.bind(this);
```
One more, for completeness 😄